CMakePresets.json

```
@@ -0,0 +1,432 @@
+        "alpaka_UBSAN": "ON"
+      }
+    },
+    {
+      "name": "cpu-base",
+      "hidden": true,
+      "cacheVariables": {
+        "alpaka_DEP_OMP": "ON"
+      }
+    },
+    {
+      "name": "rel-cpu-gcc",
+      "inherits": [
+        "release-base",
+        "cpu-base"
+      ],
+      "cacheVariables": {
+        "CMAKE_CXX_COMPILER": "g++"
+      }
+    },
+    {
+      "name": "rel-cpu-headercheck",
+      "inherits": [
+        "release-base",
+        "cpu-base"
+      ],
+      "cacheVariables": {
+        "alpaka_HEADERCHECKS": "ON"
+      }
+    },
+    {
+      "name": "dbg-cpu-gcc",
+      "inherits": [
+        "debug-base",
+        "cpu-base"
+      ],
+      "cacheVariables": {
+        "CMAKE_CXX_COMPILER": "g++"
+      }
+    },
+    {
+      "name": "dbg-cpu-gcc-asan",
+      "inherits": [
+        "debug-base",
+        "cpu-base",
+        "asan-base"
+      ],
+      "cacheVariables": {
+        "CMAKE_CXX_COMPILER": "g++"
+      }
+    },
+    {
+      "name": "dbg-cpu-gcc-tsan",
+      "inherits": [
+        "debug-base",
+        "cpu-base",
+        "tsan-base"
+      ],
+      "cacheVariables": {
+        "CMAKE_CXX_COMPILER": "g++"
+      }
+    },
+    {
+      "name": "dbg-cpu-gcc-lsan",
+      "inherits": [
+        "debug-base",
+        "cpu-base",
+        "lsan-base"
+      ],
+      "cacheVariables": {
+        "CMAKE_CXX_COMPILER": "g++"
+      }
+    },
+    {
+      "name": "dbg-cpu-gcc-ubsan",
+      "inherits": [
+        "debug-base",
+        "cpu-base",
+        "ubsan-base"
+      ],
+      "cacheVariables": {
+        "CMAKE_CXX_COMPILER": "g++"
+      }
+    },
+    {
+      "name": "dbg-cpu-clang",
+      "inherits": [
+        "debug-base",
+        "cpu-base"
+      ],
+      "cacheVariables": {
+        "CMAKE_CXX_COMPILER": "clang++"
+      }
+    },
+    {
+      "name": "dbg-cpu-clang-asan",
+      "inherits": [
+        "debug-base",
+        "cpu-base",
+        "asan-base"
+      ],
+      "cacheVariables": {
+        "CMAKE_CXX_COMPILER": "clang++"
+      }
+    },
+    {
+      "name": "dbg-cpu-clang-tsan",
+      "inherits": [
+        "debug-base",
+        "cpu-base",
+        "tsan-base"
+      ],
+      "cacheVariables": {
+        "CMAKE_CXX_COMPILER": "clang++"
+      }
+    },
+    {
+      "name": "dbg-cpu-clang-lsan",
+      "inherits": [
+        "debug-base",
+        "cpu-base",
+        "lsan-base"
+      ],
+      "cacheVariables": {
+        "CMAKE_CXX_COMPILER": "clang++"
+      }
+    },
+    {
+      "name": "dbg-cpu-clang-ubsan",
+      "inherits": [
+        "debug-base",
+        "cpu-base",
+        "ubsan-base"
+      ],
+      "cacheVariables": {
+        "CMAKE_CXX_COMPILER": "clang++"
+      }
+    },
+    {
+      "name": "rel-cpu-clang",
+      "inherits": [
+        "release-base",
+        "cpu-base"
+      ],
+      "cacheVariables": {
+        "CMAKE_CXX_COMPILER": "clang++"
+      }
+    },
+    {
+      "name": "rel-gpu-cuda",
```

```suggestion
      "name": "rel-gpu-cuda-nvcc-gcc",
```
Also a `"rel-gpu-cuda-nvcc-clang"` is missing.